### ISSUE TITLE
GraphQL error when launching KeystoneJS application

### ISSUE THREAD

**Issue by leoseccia** [keystonejs/keystone#2290]
# Bug report
GraphQL error when launching a newly created KeystoneJS application.

## Describe the bug
I just created a KeystoneJS using `yarn create keystone-app my-app`.

When I try to run it using `yarn dev` and browse to it I get the following error:

```
Error: Cannot use GraphQLSchema "[object GraphQLSchema]" from another module or realm.
```

NB. I am on Windows 10 / WSL (v1) with Ubuntu. KeystoneJS is running from Linux and MongoDB server is installed and running on Windows. This is because when I had it running in Linux, mongod showed as running and listening but I was not able to connect to it (via KeystoneJS or via shell using mongo command).

## To Reproduce

Steps to reproduce the behaviour. Please provide code snippets or a repository:

1. create a KeystoneJS using `yarn create keystone-app my-app`
2. run using `yarn dev`
3. browse to app url
4. See error in the shell

## Expected behaviour

It should just work... Instead, it crashes.

## Screenshots

![image](https://user-images.githubusercontent.com/2159624/73186654-0826c300-4118-11ea-9fcf-0e7d11421a44.png)

## System information

- OS: Windows 10 / WSL (v1) with Ubuntu. KeystoneJS is running from Linux and MongoDB server is installed and running on Windows
- Browser (if applies): Chrome

### Stackoverflow

https://stackoverflow.com/questions/59933782/graphql-error-when-launching-keystonejs-application

### Stacktrace

```
Error: Cannot use GraphQLSchema "[object GraphQLSchema]" from another module or realm.

Ensure that there is only one instance of "graphql" in the node_modules
directory. If different versions of "graphql" are the dependencies of other
relied on modules, use "resolutions" to ensure only one version is installed.

https://yarnpkg.com/en/docs/selective-version-resolutions

Duplicate "graphql" modules cannot be used at the same time since different
versions may have different capabilities and behavior. The data from one
version used in the function from another could produce confusing and
spurious results.
    at instanceOf (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/jsutils/instanceOf.js:28:13)
    at isSchema (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/type/schema.js:36:34)
    at assertSchema (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/type/schema.js:40:8)
    at validateSchema (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/type/validate.js:44:28)
    at graphqlImpl (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/graphql.js:79:62)
    at /my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/graphql.js:28:59
    at new Promise (<anonymous>)
    at graphql (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/node_modules/graphql/graphql.js:26:10)
    at _graphQLQuery.<computed> (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/lib/Keystone/index.js:477:7)
    at Keystone.executeQuery (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/lib/Keystone/index.js:252:14)
    at Object.module.exports [as onConnect] (/my/home/path/my-first-ks-app/initial-data.js:10:22)
    at /my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/lib/Keystone/index.js:323:35
    at processTicksAndRejections (internal/process/task_queues.js:97:5)
    at async executeDefaultServer (/my/home/path/my-first-ks-app/node_modules/@keystonejs/keystone/bin/utils.js:114:3)
error Command failed with exit code 1.
```

**Comment by elhakeem** [comment]
I was using `graphql@15.0.0` when I got this error.
I fixed it by downgrading to `graphql@14.6.0`.
I got this issue on a keystone project with apollo.

**Comment by MadeByMike** [comment]
Thanks for the extra info @elhakeem! I'll make a not to test this combination. For now it looks like this specific issue is solved.
